Sources/TuistKit/Services/InstallService.swift

```diff
@@ -75,5 +75,20 @@ final class InstallService {
 
             try swiftPackageManagerController.resolve(at: packageManifestPath.parentDirectory, printOutput: true)
         }
+
+        try savePackageResolved(at: packageManifestPath.parentDirectory)
+    }
+
+    private func savePackageResolved(at path: AbsolutePath) throws {
+        let sourcePath = path.appending(component: Constants.SwiftPackageManager.packageResolvedName)
+        guard fileHandler.exists(sourcePath) else { return }
```

Comment:
Should we throw in this case? The file should always exist after installation goes through.

Reply:
I was concerned about breaking a `tuist install` that actually succeeded unnecessarily, but you're right, it wouldn't actually have succeeded if there is no `Package.resolved`. Let's throw.

Reply:
Actually, if the `Package.swift` doesn't list any remote dependencies, SPM doesn't create a `Package.resolved`. So there are cases where we wouldn't want to throw, like `fixtures/app_with_previews`.